.hyperlint/.vale.ini

```diff
@@ -0,0 +1,7 @@
+StylesPath = styles
```

Comment:
This is the base style guide check cconfiguration, no action needed here on your part - this won't change much or often.